parfive/main.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+import sys
+
+from parfive import Downloader
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Parfive, the python asyncio based downloader')
+    parser.add_argument('urls', metavar='URLS', type=str, nargs='+',
+                        help='URLs of files to be downloaded.')
+    parser.add_argument('--max-conn', type=int, default=5,
+                        help='Number of maximum connections.')
+    parser.add_argument('--overwrite', action='store_const', const=True, default=False,
+                        help='Overwrite if the file exists.')
+    parser.add_argument('--no-file-progress', action='store_const', const=False, default=True, dest='file_progress',
```

Comment:
If this is "no-file-progress" should the default not be False, i.e. show progress by default?